types/node/ts3.1/globals.d.ts

```diff
@@ -577,7 +600,7 @@ declare namespace NodeJS {
         filename: string;
         loaded: boolean;
         /** @deprecated since 14.6.0 Please use `require.main` and `module.children` instead. */
-        parent: Module | null | undefined;
+        parent: Module | null;
```

Comment:
This can be [`Module | null | undefined`](https://nodejs.org/api/modules.html#modules_module_parent).
```suggestion
        parent: Module | null | undefined;
```

Reply:
The option ```undefined`` is causing this bug: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/47602

Which is why I think it should be removed.